examples/monaco-editor-webpack-plugin/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "monaco-editor-webpack-plugin-example",
+  "version": "1.0.0",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "start": "webpack serve --open --mode development",
+    "build": "webpack --mode production"
+  },
+  "dependencies": {
+    "monaco-editor": "^0.29.0",
+    "monaco-yaml": "file:../.."
+  },
+  "devDependencies": {
+    "css-loader": "^6.4.0",
+    "style-loader": "^3.3.1",
+    "monaco-editor-webpack-plugin": "^5.0.0",
+    "webpack": "^5.59.1",
+    "webpack-cli": "^4.9.1",
+    "webpack-dev-server": "^4.0.0"
+  }
```

Comment:
> Since there isn’t a real objective distinction between them for unpublished webpack packages, I prefer to not make a distinction at all.

I can't agree with you. Separation gives the user an insight which packages are build dependencies and which are runtime dependencies and will be bundled. I consider this as a good practise and form of documentation. But you're the boss, so I did it.

If npm version is important, maybe you should consider this in `package.json`?
```JSON
"engines": {
  "npm": ">=8.1.2"
}
```